### PR TITLE
Fix id anchor link

### PR DESCRIPTION
**Pull request by scottianstewart**
- change link href from `#matchers` to `#assertions`

### CODE REVIEW

File: packages/jasmine-enzyme/README.md [hunk 1]
```diff
@@ -41,36 +42,36 @@ describe('test', () => {
...
-* [toHaveRef()](#tohaveref)
-* [toHaveState()](#tohavestate)
-* [toHaveStyle()](#tohavestyle)
-* [toHaveTagName()](#tohavetagname)
-* [toHaveText()](#tohavetext)
-* [toIncludeText()](#toincludetext)
-* [toHaveValue()](#tohavevalue)
-* [toMatchElement()](#tomatchelement)
-* [toMatchSelector()](#tomatchselector)
+- [toBeChecked()](#tobechecked)
```
Comment (ljharb) [inline packages/jasmine-enzyme/README.md]: seems like a lot of unrelated changes?

Comment (scottianstewart) [reply packages/jasmine-enzyme/README.md]: Thank you! Reverted and disabled prettier
